clang/test/dpct/math_specific_UDR_test.cu

```diff
@@ -73,7 +73,7 @@ void testDouble() {
 // CHECK:   memset(hostArrayFloat, 0, bytes);
 // CHECK:   float *deviceArrayFloat;
 // CHECK:   deviceArrayFloat = (float *)sycl::malloc_device(bytes, q_ct1);
-// CHECK:   q_ct1.memcpy(deviceArrayFloat, hostArrayFloat, bytes);
+// CHECK:   q_ct1.memcpy(deviceArrayFloat, hostArrayFloat, bytes).wait();
```

Comment:
Please add lit test for following case: 
for(...)
   ...
  cudaMemcpy(deviceArrayDouble, hostArrayDouble, bytes, cudaMemcpyHostToDevice).wait();// is host ArrayDouble saft to keep;
  kernelFunc<<<1, 1>>>(deviceArrayDouble);
  free(hostArraydouble)